src/App.jsx

```diff
@@ -14,57 +17,81 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const goodsMap = goods.map(good => {
+  return {
+    name: good,
+    id: uuid(),
+  };
+});
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  function handleClick(good) {
+    setSelectedGood(good.name);
+  }
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  function removeSelection() {
+    setSelectedGood('');
+  }
```

Comment:
The method `removeSelection` does not start with a verb, which is against the guideline provided in the checklist. Consider renaming it to something like `clearSelection` to better reflect the action it performs.

Reply:
but 'remove' is a verb

Reply:
I agree with ai, clearSelection, a clarify name for the function